packages/svelte/tests/snapshot/samples/hmr/_expected/client/index.svelte.js

```diff
@@ -12,12 +12,14 @@ function Hmr($$anchor) {
 if (import.meta.hot) {
 	const s = $.source(Hmr);
 	const filename = Hmr.filename;
+	const $$original = Hmr;
 
 	Hmr = $.hmr(s);
 	Hmr.filename = filename;
+	Hmr.original = $$original;
```

Comment:
should `filename` and `original` be behind a symbol, or at least non-enumerable? Didn't occur to me until now but it's probably a bit weird that we're adding these properties for anyone to look at